Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
+      - go run main.go
+  dev_server:
```

Comment:
In the `dev_server` task, using a multiline command can reduce readability. Consider breaking it into multiple commands or using a script instead for better clarity and maintainability.